packages/video_player/video_player/example/lib/main.dart

```diff
@@ -250,17 +255,95 @@ class _BumbleBeeRemoteVideoState extends State<_BumbleBeeRemoteVideo> {
         children: <Widget>[
           Container(padding: const EdgeInsets.only(top: 20.0)),
           const Text('With remote mp4'),
+          FutureBuilder<bool>(
+            key: _pictureInPictureKey,
+            future: _controller.isPictureInPictureSupported(),
+            builder: (BuildContext context, AsyncSnapshot<bool> snapshot) =>
+                Text(snapshot.data ?? false
+                    ? 'Picture-in-picture is supported'
+                    : 'Picture-in-picture is not supported'),
+          ),
+          Row(
+            children: <Widget>[
+              const SizedBox(width: 16),
+              const Expanded(
+                child: Text(
+                    'Start picture-in-picture automatically when going to background'),
+              ),
+              Switch(
+                value: _enableStartPictureInPictureAutomaticallyFromInline,
+                onChanged: (bool newValue) {
+                  setState(() {
+                    _enableStartPictureInPictureAutomaticallyFromInline =
+                        newValue;
+                  });
+                  _controller.setAutomaticallyStartsPictureInPicture(
+                      enableStartPictureInPictureAutomaticallyFromInline:
+                          _enableStartPictureInPictureAutomaticallyFromInline);
+                },
+              ),
+              const SizedBox(width: 16),
+            ],
+          ),
+          MaterialButton(
+            color: Colors.blue,
+            onPressed: () {
+              final RenderBox? box =
+                  _playerKey.currentContext?.findRenderObject() as RenderBox?;
+              if (box == null) {
+                return;
+              }
+              final Offset offset = box.localToGlobal(Offset.zero);
+              _controller.setPictureInPictureOverlaySettings(
+                settings: PictureInPictureOverlaySettings(
+                  rect: Rect.fromLTWH(
+                    offset.dx,
+                    offset.dy,
+                    box.size.width,
+                    box.size.height,
+                  ),
+                ),
+              );
+            },
+            child: const Text('Set picture-in-picture overlay rect'),
```

Comment:
Why is this something that clients have to call manually? Shouldn't the controller be determining this information from the widget as needed?